.travis.yml

```diff
@@ -1,5 +1,5 @@
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
why is sudo required?

Reply:
This is 'preventative' - it increases the container RAM from 4GB to 7.5GB. In the past I've seen problems with the Travis memcap when running multiple instances of testrpc  because there's very poor garbage collection there. And solidity-coverage has a relatively large memory footprint. It *may* not be necessary - I think the only drawback is that it takes slightly longer for the container to boot. [Info about this here.](https://docs.travis-ci.com/user/reference/overview/). Let me know if you think this should be undone.

Reply:
Nice, thanks for adding that info, I didn't know about this. It might explain why our builds fail sporadically with issues connecting to `testrpc` which was probably because of OOM. Let's keep it :+1: